wiki/Client/Beatmap_editor/Compose/en.md

```diff
@@ -1,351 +1,212 @@
+![](img/beat-snap-divisor.jpg "Beat snap divisor")
 
-#### Divisor 1/6
-
-![Divisor 1/6](/wiki/shared/BSD_1_6.jpg "Divisor 1/6")
-
-- Double Triplets*
-- Has two purple ticks between each red tick
-- Compound timing
-
-#### Divisor 1/8
-
-![Divisor 1/8](/wiki/shared/BSD_1_8.jpg "Divisor 1/8")
-
-- Eighth Beats
-- Adds yellow ticks, in addition to the red and blue
-
-**Insert Break Time** button will start/stop "Break Time" in current timestamp and **x/y** is current mouse coordinate.
-
-### Centre-Left (Selector)
-
-![Hit Notes Selector](img/BE_HO.jpg "Hit Notes Selector")
-
-**Your construction toolkit, made for DIY (do it yourself).**
+| Name | Description |
+| :-- | :-- |
+| [Beat Snap Divisor](/wiki/Client/Beatmap_editor/Beat_snap_divisor) | Limits where hit objects can be placed so that they always fall on the beat. Moving the slider to the right increases the granularity of the hit objects. |
+| `Insert Break Time` | This button will start/stop [break](/wiki/Beatmap/Break) time in current timestamp. |
+| x/y | The current mouse coordinate. |
 
-**Please note that in osu!, default note hitsounds work differently than in osu!mania.**
+Hold `Alt` to trigger **Distance Spacing**. The value can range from 0.1x to 6.0x.
 
-- Adding clap on a notes will play `hitclap` + `hitnormal` sound on hit while in osu!mania: will only play `hitclap` sound on hit
+### Selector
 
-#### Selector
+![](img/tools.jpg "Hit objects selector")
 
-| Buttons / Header (Keyboard Shortcut) | Usage | Description |
+| Buttons (Keyboard shortcut) | Usage | Description |
 | :-- | :-- | :-- |
-| Sampleset | Auto, Normal, Soft, Drum | This ignore the timing section's Sampleset setting , and add the selected one... e.g. Timing point with **soft 77%**, While this Sampleset set into **Drum** and have **Clap** on the notes, will play default drum-hitnormal 77% + drum-hitclap 77% on hit |
-| Additions | Auto, Normal, Soft, Drum | Additions hitsound means add Clap, Finish, and Whistle with the SELECTED set... e.g. Sampleset **Normal**, Timing point **Soft 77%**, while Additions is set to **Drum** and have **Clap** on the notes, will result of normal-hitnormal 77% + drum-hitclap 77% on hit |
-| Select (`1`) | **Left-click/drag:** Select/Adjust the notes/sliderpoints location. **Right-click:** Remove notes/sliderpoints. **`Ctrl` + Click:** Multiple selection. | Select and modify existing notes. |
-| Circle (`2`) | **Left/Right-click:** Add/Remove hit circle. | Hit Circle. |
-| Slider (`3`) | **Left/Right-click:** Start/End Slider. Add/Remove sliderpoints (white square). **Left-Click x2:** New curve section (red square). | Slider. |
-| Spinner (`4`) | **Left/Right-click:** Start/End Spinner at current timestamp. | Spinner. Make sure Auto can at least score 2,000 Bonus per Spinner (to prevent the infamous spinner spam/ninja spinner). |
+| `Sampleset` | Auto, Normal, Soft, Drum | This ignores the timing section's sampleset setting, and adds the selected one. |
```

Comment:
I feel that leaving the usage column blank here is a better choice, as these aren't exactly actions
```suggestion
| `Sampleset` | | Override the [sampleset](/wiki/Beatmapping/Sampleset) of selected objects (including their hitnormals). Choosing `Auto` resets the sampleset to that of an active [timing point](/wiki/Client/Beatmap_editor/Timing#timing-points). |
```